service-front/public/assets/v2/css/download-message.css

```diff
@@ -0,0 +1,8 @@
+.download-message > h2 {
```

Comment:
As this is a generated file (I think), it shouldn't be checked in.

Reply:
Thanks! I didn't realise that we wouldn't check these in, will remove them

Reply:
Oh wait, but it's an individual css file that's minified like application.js, so if I didn't check it in, it'd autogenerate locally each time no?

Reply:
I've just cleaned out a few files that I hadn't deleted after a file name change

Reply:
> Oh wait, but it's an individual css file that's minified like application.js, so if I didn't check it in, it'd autogenerate locally each time no?

Yes, it will autogenerate locally, providing it's mentioned in the Gruntfile.